### DISCUSSION TITLE
Can't use update OTA - it doesn't see any new version

### DISCUSSION THREAD

**Discussion by qubiack** [RavenSystem/esp-homekit-devices#1745] [General]
When I open a setup page I don't see any version to update as on the presented image and I can't update my Shelly 1 OTA. I tried to use also HAA Manager from AppStore but both - update devices & update all devices have the same issue and I can't update them.

![image](https://user-images.githubusercontent.com/19862364/152702294-46209c56-a8f9-4c43-b294-f28a1af8107f.png)

**Comment by i3laze** [comment]
I guess it’s your old OTA firmware doesn’t support recent GitHub changes, like partial download.

Please try my custom HTTP server, which should allow you to bump to OTA v5.8.2:
91.218.86.30 port 8888 no secure

Then you can remove custom server setting and update to latest v10+ from GitHub.

**Comment by RavenSystem** [comment]
Please, use Discussions section and leave Issues section only for issues. Thanks!

https://github.com/RavenSystem/esp-homekit-devices/wiki#issues

**Comment by ghost** [comment]
I'm having the same issue. My Sonoff Mini and Sonoff TX has firmware Home Accessory Architect v10.2.0 Kestrel and in the setup page I don't see any OTA version to update to. Above solution doesn't work for my devices. Would you be able to give any further advice?

**Comment by PedroDeLaFrance** [comment]
Same problem.


![Capture d’écran 2023-08-08 143140](https://github.com/RavenSystem/esp-homekit-devices/assets/140181697/54e8e341-c697-4c95-b8bf-fa8790e43006)

Impossible to update OTA my SONOFF MINI R2.

I test Auto Update, HAA Manager iOS too...

**Reply by i3laze** [reply]
Please post a screenshot of the lower part of settings screen. I wonder if you specified any custom HTTP server/port.

**Reply by PedroDeLaFrance** [reply]
I don't have specified any custom HTTP server/port, I think.
![Capture web_9-8-2023_104747_192 168 1 149](https://github.com/RavenSystem/esp-homekit-devices/assets/140181697/42b99525-b561-4252-bbf2-c772f6f0ecf6)
